.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,92 @@
+name: "🐛 Bug report"
+description: Report a bug found while using the Berty apps or packages.
+labels: ["bug", "Ready for Review"]
+assignees:
+- peletron
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: dropdown
+    id: product
+    attributes:
+      label: Berty product
+      description: What Berty product are you seeing the problem on?
+      options:
+        - Mobile app
+        - Desktop app
+        - CLI tools (berty mini, daemon, etc.)
+        - Code package
+    validations:
+      required: true
+  - type: input
+    id: product-version
+    attributes:
+      label:  Berty product version
+      description: What is the version of this Berty product?
+      placeholder: You can find the version of the mobile and desktop apps in the settings. To display the CLI tool version, run `berty version`.
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: OS
+      description: What OS are you seeing the problem on?
+      multiple: true
```

Comment:
I was thinking about this as well. The reason why I set this as `multiple: true` is that you have some beta-testers that actively reporting bugs for the platform. Since this is an open-source project there will be people who can check bugs on multiple devices in order to provide better reports and contribute to the project, so somebody might check bugs on Android and iOS and then report. Does this makes sense?

Reply:
Yes, but if they choose multiple OS, how do they carry over multiple OS versions, multiple devices, etc.?
I think it's going to make things more complex. Might as well let the user enter a single OS (the most relevant one) and specify in the Other section that they also see the concern elsewhere. 
On our side we can apply the right labels on the issue with the different OS concerned.

In my opinion the best option is to disable the multiple.